apiv2/views.py

```diff
@@ -983,6 +1003,27 @@ def post(self, request,  *args, **kwargs):
 # OTHER VIEWS
 #############
 
+
+@api_view(['GET'])
+@throttle_classes([])
+@authentication_classes([])
+@permission_classes([])
+def download_from_token(request, token):
+    try:
+        token_contents = jwt.decode(token, settings.SECRET_KEY, leeway=10)
+    except jwt.ExpiredSignatureError:
+        raise UnauthorizedException(msg="This token has expried.")
+    except jwt.InvalidTokenError:
+        raise UnauthorizedException(msg="Invalid token.")
+    try:
+        sound = Sound.objects.get(id=token_contents.get('sound_id', None))
```

Comment:
sure, never got to be consistent with this. I always try to use single quote, then some strings come which have single quotes inside and I try to continue using single quote and backslash the quotes inside. Then  I got tired and I switch to double quote again. Then I just forget about what did I decide...